app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<header class="login-header">
+<%= link_to root_path do %>
+<i class=" fa fa-arrow-left"></i>
+<% end %>
+<div class="login-link">
+<p>REGISTER</p>
+</div>
+<p class="p-hide">Next</p>
+</header>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+<div class="cont-top">
```

Comment:
- [ ] please make sure your application layout and distance between elements is the same as in the template provided on Behance:
![image](https://github.com/hmunish/ror-capstone/assets/39240820/532a2f87-875e-4a6e-9484-b8fb037e3802)

Reply:
@mikethreels, thank you for the code review. I am a bit confused about which layout and distance seems to be inappropriate & what changes needs to be done.

Reply:
@mikethreels , please can you explain a bit further.